spec/lib/sports_data_api/ncaafb/game_spec.rb

```diff
@@ -0,0 +1,157 @@
+require 'spec_helper'
+
+describe SportsDataApi::Ncaafb::Game, vcr: {
+    cassette_name: 'sports_data_api_ncaafb_game',
+    record: :new_episodes,
+    match_requests_on: [:host, :path]
+} do
+  let(:season) do
+    SportsDataApi.set_key(:ncaafb, api_key(:ncaafb))
+    SportsDataApi.set_access_level(:ncaafb, 't')
+    SportsDataApi::Ncaafb.schedule(2014, :REG)
+  end
+
+  let(:boxscore) do
+    SportsDataApi.set_key(:ncaafb, api_key(:ncaafb))
+    SportsDataApi.set_access_level(:ncaafb, 't')
+    SportsDataApi::Ncaafb.boxscore(2014, :REG, 10, 'IOW', 'NW')
```

Comment:
Should have been `SportsDataApi::Ncaafb.boxscore(2013, :REG, 8, 'LOU', 'UCF')`. Other than that looks good :+1: 

related: [Blake Bortles' Game Winning Drive vs. Louisville](https://www.youtube.com/watch?v=AsHSYdNv2C4)